code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -386,6 +386,23 @@
 			return 2
 	return ..()
 
+///Runs the drone's inbuilt anti-grab measures, shocking the grabber after a while
+/mob/living/critter/flock/drone/proc/do_antigrab()
+	//if we're blocking that means we're not grabbed
+	if (!length(src.grabbed_by) || src.find_type_in_hand(/obj/item/grab/block))
+		src.antigrab_counter = 0
+		return
+
+	antigrab_counter++
+	if (antigrab_counter >= src.antigrab_fires_at)
```

Comment:
```suggestion
	src.antigrab_counter++
	if (src.antigrab_counter >= src.antigrab_fires_at)
```
Very minor, not necessary, but was kind of using some interchanging

Reply:
shame on me smh